larray/core/array.py

```diff
@@ -6944,45 +7063,46 @@ def plot(self):
 
         Examples
         --------
-        >>> import matplotlib.pyplot as plt # doctest: +SKIP
-        >>> a = ndtest('gender=M,F;age=0..20')
+        >>> import matplotlib.pyplot as plt                                    # doctest: +SKIP
```

Comment:
putting `# doctest: +SKIP` should be required if the `setup.cfg` file is updated so that `Array.plot` is excluded.
The line to add should be something like:
```
--deselect larray/core/array.py::larray.core.array.Array.plot
```

Reply:
Maybe duplicate the tests below as a new test_plot() function and create a `@needs_matplotlib` tag?

Reply:
The problem is how to actully test the result. We could test the created .png file is byte-for-byte equal but then I fear it would break when upgrading matplotlib. Or do you mean just running the test to make sure it does not crash?

Reply:
In other words, forget about that "byte-for-byte equal" test on the PNG file.